app/views/admin/people/show.html.erb

```diff
@@ -1,45 +1,48 @@
-<% page_title @person.name %>
+<% content_for :page_title, @person.name %>
+<% content_for :title, @person.name %>
+<% content_for :title_margin_bottom, 2 %>
 
-<%= content_tag_for(:div, @person) do %>
-  <div class="person-header">
-    <h1><%= @person.name %></h1>
-    <%= view_on_website_link_for @person %>
+<p class="govuk-body govuk-!-margin-bottom-8"><%= view_on_website_link_for @person, class: "govuk-link", target: "blank" %></p>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "components/secondary_navigation", {
+      aria_label: "Document navigation tabs",
+      items: secondary_navigation_tabs_items(current_user, @person, request.path)
+    } %>
   </div>
+</div>
+
 
-  <section class="person-details">
-    <%= tab_navigation_for(@person) %>
-
-    <article class="person">
-
-      <% if @person.image_url %>
-        <%= image_tag @person.image.url(:s300) %>
-      <% end %>
-
-      <div class="roles">
-        <h2><%= @person.current_role_appointments.collect(&:role_name).to_sentence %></h2>
-      </div>
-
-      <div class="biography">
-        <h2>Biography</h2>
-        <div class="content row add-bottom-margin">
-          <div class="col-md-8">
-            <pre><%= @person.biography %></pre>
-          </div>
-        </div>
-      </div>
-    </article>
-
-    <div class="form-actions">
-      <%= link_to "Edit", [:edit, :admin, @person], class: "btn btn-lg btn-primary"%>
-      <% if @person.destroyable? %>
-        <%= button_to 'Delete',
-              admin_person_path(@person),
-              method: :delete,
-              class: "btn btn-lg btn-danger",
-              data: { confirm: 'Are you sure you want to delete this person?' } %>
-      <% else %>
-        <span><em>Note: This person cannot be deleted as they are currently assigned to a role</em></span>
-      <% end %>
-    </div>
-  </section>
+<% if @person.image_url %>
+  <div class="">
+    <%= image_tag @person.image.url(:s300) %>
+  </div>
 <% end %>
+
+<div class="roles">
+  <h2 class="govuk-heading-l"><%= @person.current_role_appointments.collect(&:role_name).to_sentence %></h2>
```

Comment:
```suggestion
  <%= render "govuk_publishing_components/components/heading", {
    text: @person.current_role_appointments.collect(&:role_name).to_sentence,
    font_size: "l"
  } %>
```

I would recommend using the component to keep it consistent and also allow for easier upgrades should the styling of headings change in the future.